curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-059.md

```diff
@@ -0,0 +1,282 @@
+---
+id: 6201a1a7af32c287bd6b8183
+title: Step 59
+challengeType: 0
+dashedName: step-59
+---
+
+# --description--
+
+Select your `td` elements with the `class` value of `current`, and make the font italic.
+
+# --hints--
+
+You should have a `td.current` selector.
```

Comment:
should `td[class="current"] {}` be accepted too here? there is no specification of which method to use

Reply:
Sure!